jest.setup.tsx

```diff
@@ -51,3 +50,15 @@ jest.mock('next/navigation', () => ({
   }),
   usePathname: () => '',
 }))
+
+jest.mock('@/publicodes-state', () => ({
```

Comment:
Should be specific at the test case. Otherwise how would you simulate a user with a previous simulation ?

Reply:
J'ai créé un wrapper qui me permet de modifier ça > wrapper.tsx